### PR TITLE
WL-4160 Changed the colour of the login table for 'xlogin.vm' file.

### PR DESCRIPTION
**Pull request by ouit0408**

### CODE REVIEW

File: library/src/webapp/skin/neo-ox/tool.css [hunk 1]
```diff
@@ -1019,9 +1019,10 @@ tr.selectedFound td {
     font-weight: normal;
 }
 .login {
-    border: 1px solid #dedede;
```
Comment (buckett) [inline library/src/webapp/skin/neo-ox/tool.css:1022]: Isn't this used in other places than just the login page?

Comment (ouit0408) [inline library/src/webapp/skin/neo-ox/tool.css:1022]: It is used at two other places.I will make changes in the file and not in this css.

**Comment by ouit0408** [comment]
Since the class .login was used at other places too , I have added new class '.oulogin' to the table in xlogin.vm file. Added colour to this new class 'oulogin' for the table. 
New pull request https://github.com/ox-it/wl-login/pull/6 . sorry about this my fault i didn't check the references for the class.

**Comment by buckett** [comment]
Closing as we're not doing the move to 960px.

**Pull request closed without merging**